client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.html

```diff
@@ -23,143 +23,172 @@
+                    <mat-header-cell *matHeaderCellDef mat-sort-header>Speakers</mat-header-cell>
+                    <mat-cell *matCellDef="let motion">
+                        <button mat-icon-button (click)="onSpeakerIcon(motion, $event)" [disabled]="isMultiSelect">
+                            <mat-icon
+                                [matBadge]="motion.agendaSpeakerAmount > 0 ? motion.agendaSpeakerAmount : null"
+                                matBadgeColor="accent"
+                            >
+                                mic
+                            </mat-icon>
+                        </button>
+                    </mat-cell>
+                </ng-container>
 
-        <mat-header-row *matHeaderRowDef="getColumnDefinition()"></mat-header-row>
-        <mat-row
-            [ngClass]="selectedRows.indexOf(row) >= 0 ? 'selected' : ''"
-            (click)="selectItem(row, $event)"
-            *matRowDef="let row; columns: getColumnDefinition()"
-            class="lg"
-        >
-        </mat-row>
-    </mat-table>
+                <mat-header-row *matHeaderRowDef="getColumnDefinition()"></mat-header-row>
+                <mat-row
+                    [ngClass]="selectedRows.indexOf(row) >= 0 ? 'selected' : ''"
+                    (click)="selectItem(row, $event)"
+                    *matRowDef="let row; columns: getColumnDefinition()"
+                    class="lg"
+                >
+                </mat-row>
+            </mat-table>
 
-    <mat-paginator class="on-transition-fade" [pageSizeOptions]="pageSize"></mat-paginator>
+            <mat-paginator class="on-transition-fade" [pageSizeOptions]="pageSize"></mat-paginator>
+        </span>
+        <span *ngSwitchCase="'tiles'">
+            <os-grid-layout title="{{ 'Favorites' | translate }}">
+                    <os-block-tile (clicked)="navigateTo(null, true)" [orientation]="'horizontal'" [block]="'Fav'" [blockType]="'text'" title="{{ 'Favorite motions' | translate}}">
+                        <ng-template #contentNode>
+                            <div translate>
+                                Containing motions: <mat-chip>{{ motionsAsFavorite().length }}</mat-chip>
+                            </div>
+                        </ng-template>
+                    </os-block-tile>
+                </os-grid-layout>
+            <os-grid-layout title="{{ 'All categories' | translate }}">
+                <os-block-tile *ngFor="let category of categories" (clicked)="navigateTo($event)" [orientation]="'horizontal'" [block]="category.prefix" [blockType]="'text'" [data]="category" [title]="category.name">
+                    <ng-template #contentNode>
+                        <div translate>
+                            Containing motions: <mat-chip>{{ motionsInCategory(category).length }}</mat-chip>
+                        </div>
+                    </ng-template>
+                </os-block-tile>
+            </os-grid-layout>
+        </span>
```

Comment:
We may also need a `no category` tile

Reply:
I've built also a component `tile.component`, where `block-tile.component` is a subclass of it. The `tile.component` has the general functionality to fit the layout.
The `block-tile.component` is a little specific with two separated parts: The header ("block") and the content.

Does this answer your question? ;)